src/app.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="dark">
```

Comment:
This will work, but it won't work with the light/dark picker, please check the system for light/dark mode switching and adapt it accordingly

Reply:
As agreed, I added a small red shadow (in the spirit of the Stak logo) on the snowflakes so that they become more noticeable in the light mode.